packages/pass-style/package.json

```diff
@@ -19,6 +19,7 @@
   "module": "./index.js",
   "exports": {
     ".": "./index.js",
+    "./src/types.js": "./src/types.js",
```

Comment:
Interesting. Do you expect this precedent to spread?

Reply:
I'd rather export the types at the top level but I think that will require either manually maintaining an `index.d.ts` next to the `index.js` or having a single `index.ts` that can export both JS code and TS types. But the latter would entail a build step and thus require some new workflows and buy-in from the repo maintainers

Reply:
I'm confused, why can't `index.js` do `export * from './src/types.js'` ?

Reply:
yeah, I realized that soon after my comment. it does require an eslint suppression but I'll do that